src/validations/passport-gated/schema.json

```diff
@@ -138,19 +139,23 @@
             {
               "const": "OR",
               "title": "Require at least one stamp"
+            },
+            {
+              "const": "",
+              "title": "Any (Stamps MUST be empty)"
             }
           ]
         },
         "scoreThreshold": {
           "type": "number",
           "title": "Passport Score Threshold",
-          "description": "Minimum Passport score threshold (>) to be eligible as a valid Passport",
+          "description": "Minimum Passport score threshold (>=) to be eligible as a valid Passport",
           "minimum": 0,
           "maximum": 100,
           "default": 0
         }
       },
-      "required": ["stamps", "operator"],
+      "required": ["scoreThreshold"],
```

Comment:
What exactly is `scoreThreshold` and why is it required now?

Reply:
`scoreThreshold` is the minimum score a Passport must fulfil to become eligible. The Passport team requested to change the required parameters for this strategy, I think to cover cases where a user can have any valid stamp (>0). Before, the only options were `at least one (>=1) (OR operand)` and `all stamps (m == n) (AND operand)`. @0xZakk could give a more detailed explanation of why they're looking to implement these changes

Reply:
When you say "any valid stamp (>0)," does that mean they're looking to include cases where, say, 2 out of 5 stamps would make you eligible? Would love to get a bit more clarity on this, maybe @0xZakk can weigh in?

Reply:
Very sorry for the delayed response. 

It's a best practice to require a score threshold as opposed to specific Stamps, so we made the recommendation for this update. 

Requiring a specific Stamp will not work for all use cases, especially if there isn't a specific Stamp that makes sense for their use case. Because of this, we suggested that we make any Stamp selection optional. 

The 'require at least one Stamp' is not something we've recommended in the past, but I could see a potential use case there. I would guess that this option will rarely be used.

Reply:
This is to ensure the transition of the validation, so that spaces which are using stamps at the minute will not see an error caused by lack of minimum threshold set (which is a new param for the validation). 

Is that correct @lebraat ?

Reply:
Truth. We don't want any of the early adopters to experience any errors. 

Regardless, risk is low that this will cause much user confusion/frustration considering it's still marked as 'beta'. We have held off on doing a substantial marketing push until this PR is merged.

Is there a way for you all to pull a list of Spaces that are utilizing this functionality?

Reply:
Hi @lebraat Sorry for the delayed response, right now there is no way to get spaces with validation but you can query proposals using our API https://hub.snapshot.org/graphql

```graphql
{
  proposals(first: 1000, where: { validation: "passport-gated" }) {
    title
    id
    space {
      id
    }
    validation {
      name
      params
    }
    created
  }
}
```

Reply:
These changes look's good to me, we are just worried about space's questions in our support 😄 but we can go with this if you think you had enough marketing push, let me know.

Reply:
Feel free to send support messages my way so I can get a good understanding of what those look like. I can also work with you all to make adjustments to the docs to make things more clear, but I believe the updates in this PR will make the integration much more intuitive for users.